Module14/03_sum_and_dif/main.py

```diff
@@ -0,0 +1,24 @@
+def digits_amount(num):
+    counter = 0
+    while num:
+        num //= 10
+        counter += 1
+    print("Amount of digits in number:", counter)
+    return counter
+
+
+def digits_sum(num):
+    sum_ = 0
+    while num:
+        sum_ += num % 10
+        num //= 10
+    print("Sum of digits in number:", sum_)
+    return sum_
+
+
+number = int(input("Enter the number: "))
+if number < 0:
+    print("Error!")
+else:
+    print("Difference of sum and amount of digits", digits_sum(number) -
```

Comment:
function must do only 1 thing. Your function printing value and returning value that also printing elsewhere.
Refactor this, but without any print usage inside function

Reply:
done